src/bin/init.rs

```diff
@@ -0,0 +1,54 @@
+use std::env;
+
+use cargo::ops;
+use cargo::util::{CliResult, CliError, Config};
+
+#[derive(RustcDecodable)]
+struct Options {
+    flag_verbose: bool,
+    flag_quiet: bool,
+    flag_color: Option<String>,
+    flag_bin: bool,
+    flag_name: Option<String>,
+    flag_vcs: Option<ops::VersionControl>,
+}
+
+pub const USAGE: &'static str = "
+Create a new cargo package in current directory
+
+Usage:
+    cargo init [options]
```

Comment:
Found out: the main thing that it's a special library (Docopt) is in use and the usage string matters.